### PR TITLE
[MM-16244] Add Elasticsearch SkipTLSVerification parameter to the config

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Ticket Link
https://mattermost.atlassian.net/browse/MM-16244

#### Related Pull Requests
- [Has enterprise changes](https://github.com/mattermost/enterprise/pull/458)
- [Has webapp changes](https://github.com/mattermost/mattermost-webapp/pull/2968)

### CODE REVIEW

**Comment by jespino** [comment]
maybe makes sense to add this to diagnostics.go?

**Comment by mgdelacroix** [comment]
@jespino you're right! Done :)

**Comment by cpanato** [comment]
will spin a test server just to validate mattermod, you can ignore thanks

File: app/diagnostics.go [hunk 1]
```diff
@@ -570,6 +570,7 @@ func (a *App) trackConfig() {
 		"live_indexing_batch_size":          *cfg.ElasticsearchSettings.LiveIndexingBatchSize,
 		"bulk_indexing_time_window_seconds": *cfg.ElasticsearchSettings.BulkIndexingTimeWindowSeconds,
 		"request_timeout_seconds":           *cfg.ElasticsearchSettings.RequestTimeoutSeconds,
+		"skip_TLS_verification":             *cfg.ElasticsearchSettings.SkipTLSVerification,
```
Comment (grundleborg) [inline app/diagnostics.go]:
```suggestion
		"skip_tls_verification":             *cfg.ElasticsearchSettings.SkipTLSVerification,
```

For consistency with everything else being lower case.

**Comment by amyblais** [comment]
@mgdelacroix Do you have more details on why this was added to v5.13?

**Pull request merged**